hashring/rbtree_test.go

```diff
@@ -567,13 +567,62 @@ func TestDelete(t *testing.T) {
 	// EMPTY TREE!
 	assert.Nil(t, tree.root, "tree root is nil")
 
-	assert.False(t, tree.Delete(1), "success, expected deletion to fail with empty tree")
+	assert.False(t, tree.Delete(1, "one"), "success, expected deletion to fail with empty tree")
 	assert.Equal(t, 0, tree.Size(), "expected tree to be empty")
 	height, err = validateRedBlackTree(tree.root)
 	assert.NoError(t, err, "expected tree to be a valid red black tree")
 	assert.Equal(t, 1, height, "expected tree to have black height of 1")
 }
 
+func TestCollision(t *testing.T) {
+	tree := makeTree()
+
+	var str string
+	var ok bool
+
+	// "seven"<"seven-two" so "seven" remains in tree
```

Comment:
seven and seven-two are both in the tree, right?
seven-two is not searchable, but I think this comment in the test is confusing.